src/fciqmc_col.jl

```diff
@@ -328,28 +346,28 @@ function fciqmc_col!(
 
     # Diagonal step:
     new_val = (1 + dτ*(shift - diagonal_element(ham, add))) * val
-    w[add] += new_val
+    threshold_projected_deposit!(s, w, add, new_val, add => val)
 
     hops = offdiagonals(ham, add)
     if s.rel_threshold * val ≥ length(hops) || val ≥ s.abs_threshold
         # Exact multiplication when conditions are met.
         factor = dτ * val
         for (new_add, mat_elem) in hops
-            w[new_add] -= factor * mat_elem
+            threshold_projected_deposit!(s, w, new_add, -factor * mat_elem, add => val)
             spawns += 1
         end
-        return (1, 0, spawns, 0, 0)
+        return (1, 0, spawns)
```

Comment:
```suggestion
        return (1, 0, length(hops))
```
This should be saving a few operations. Not sure it will make a noticeable difference, though.